packages/@guardian/source-react-components/src/icons/SvgArrowCircleClockwise.tsx

```diff
@@ -0,0 +1,16 @@
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { iconSize } from '@guardian/source-foundations';
+import type { IconProps } from './types';
+
+export const SvgArrowCircleClockwise = ({
+	size,
+}: IconProps): EmotionJSX.Element => (
+	<svg
+		// Really this is 24×24, hence the offset
```

Comment:
i know there are plans to add new sized icons, probably distinguished somehow from the current ones.

is this one of the new sized ones?

Reply:
Yes, exactly! However, I’ve made sure it falls in line with the others and assumes a 30×30 canvas size.

I think it would be possible to author most existing icons to be 24×24 by giving them a `viewBox="3 3 24 24"`, but I’m not entirely convinced that none of them goes beyond that margin. It’s generally much easier to work with icons that have tight margins.

I would love to help getting this out and automating as much of the process as possible.

Reply:
if you need this quickly, i'd recommend making it the same size/technique as the existing ones, and then we can revisit it at the same time as the others. this stuff is definitely coming up but we are not clear yet how it will emerge into the world. keeping everything consistent will (probably) make that work clearer

Reply:
Just to be clear: I made this icon so that it works exactly as the others. However, the original canvas for this icon is 24×24, so I’ve modified the `viewBox` to make sure it’s identical to all the other icons. If you add 3px padding all around a 24×24 icon, you get a 30×30.

Would you rather I’d wrap the path inside a group that translate it by `(3,3)`? It felt more natural to edit the `viewBox`.